grafana/terraform/versions.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
```

Comment:
Terraform actually made me do this 😅 

It complained about not having something specified, and told me to run `terraform 0.13upgrade`. When I did that, this is where it put the config. So that kind of implies this is kind of a "standard" place to put them.